forc-pkg/src/manifest.rs

```diff
@@ -494,14 +496,15 @@ impl PackageManifest {
     /// If `core` and `std` are unspecified, `std` will be added to the `dependencies` table
     /// implicitly. In this case, the git tag associated with the version of this crate is used to
     /// specify the pinned commit at which we fetch `std`.
-    pub fn from_file(path: &Path) -> Result<Self> {
+    pub fn from_file<P: AsRef<Path>>(path: P) -> Result<Self> {
```

Comment:
I'm also curious why this is changed

Reply:
This was a minor change, I can roll it back, but it makes easier to call any `Path`. It basically will follow what the std library does whenever it deals with paths and it is easier to call the API, because a `&str` or a `String` will match that requirement. Should I roll it back @sdankel ?

Reply:
No this is fine, thanks for explaining!